src/sensai/featuregen.py

```diff
@@ -24,28 +24,45 @@ class FeatureGenerator(ABC):
     from an input DataFrame
     """
     def __init__(self, categoricalFeatureNames: Sequence[str] = (),
-            normalisationRules: Sequence[data_transformation.DFTNormalisation.Rule] = (), addCategoricalDefaultRules=True):
+                 normalisationRules: Sequence[data_transformation.DFTNormalisation.Rule] = (),
+                 generatedColumnsRuleTemplate: data_transformation.DFTNormalisation.RuleTemplate = None, addCategoricalDefaultRules=True):
         """
         :param categoricalFeatureNames: if provided, will ensure that the respective columns in the generated data frames will
             have dtype 'category'.
             Furthermore, presence of meta-information can later be leveraged for further transformations, e.g. one-hot encoding.
         :param normalisationRules: Rules to be used by DFTNormalisation (e.g. for constructing an input transformer for a model).
             These rules are only relevant if a DFTNormalisation object consuming them is instantiated and used
             within a data processing pipeline. They do not affect feature generation.
+        :param generatedColumnsRuleTemplate: If given, a normalisation rule based on this template will be created for all generated columns.
```

Comment:
Parameter should be called "normalisationRuleTemplate".
"This parameter can be supplied instead of normalisationRules for the case where there shall be a single rule that applies to all columns generated by this feature generator."

Reply:
Done